src/internal/storage/fileset/unordered_writer.go

```diff
@@ -24,7 +23,6 @@ type UnorderedWriter struct {
 	ids                        []ID
 	getParentID                func() (*ID, error)
 	validator                  func(string) error
-	maxFanIn                   int
```

Comment:
Any context on why we're deleting the `maxFanIn`?

Reply:
It was unused. It should have been removed when compaction was removed from the unordered writer.